pom.xml

```diff
@@ -34,7 +34,7 @@
     </properties>
 
     <scm>
-        <connection>scm:git:git@github.com:jenkinsci/build-name-setter-plugin.git</connection>
+        <connection>scm:git:https://github.com/jenkinsci/build-name-setter-plugin.git</connection>
```

Comment:
Why https is better than git ?

Reply:
Github has deprecated one of the unauthenticated access protocol (i.e git:// protocol). Hence using https:// will be a better choice. For more info you can refer https://github.blog/2021-09-01-improving-git-protocol-security-github/